From 52c4ff1aa238177b0ac5c42b8308594921dbadfa Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 1 Apr 2011 19:42:33 +0800
Subject: [PATCH 21/69] ENGR00140702 Ath3k: tune config to support better file
 transfer.

tune config to support receive file from host.
without this change, BT can't receive file from sender.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 firmware/ar3k/30101/PS_ASIC.pst | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/firmware/ar3k/30101/PS_ASIC.pst b/firmware/ar3k/30101/PS_ASIC.pst
index b68b5d0..9f4f612 100644
--- a/firmware/ar3k/30101/PS_ASIC.pst
+++ b/firmware/ar3k/30101/PS_ASIC.pst
@@ -1,11 +1,11 @@
 
 // Coex Configuration
-#
-[H:S]0017
-[H:S]0022
-[H:A]d1 d1 20 00 02 01 02 09 0c 00 24 0f 04 00 00 01 
-     00 01 00 01 01 01 01 01 01 00 00 00 01 00 00 00 
-     d2 d2 
+//#
+//[H:S]0017
+//[H:S]0022
+//[H:A]d1 d1 20 00 02 01 02 09 0c 00 24 0f 04 00 00 01 
+//     00 01 00 01 01 01 01 01 01 00 00 00 01 00 00 00 
+//     d2 d2 
 // Radio table TAG 
 #
 [H:S]012C
-- 
1.8.0

